styles/base/profile-list.mcss

```diff
@@ -0,0 +1,105 @@
+ProfileList {
+  a.profile {
+    display: flex
+    padding: 4px
+    font-size: 110%
+    margin: 4px 0
+    position: relative
+    text-decoration: none
+    background-repeat: no-repeat
+    background-position: right
+    -more {
+      padding-top: 10px
+      padding-bottom: 10px
+      div.main {
+        div.name {
+          font-weight: normal
+        }
+      }
+    }
+    -following {
+      background-image: svg(following)
+    }
+    -connected {
+      background-image: svg(connected)
+    }
+    @svg connected {
```

Comment:
The only background issue I've noticed is that the _connected_ indicator isn't showing up on the "Connected Pubs" list.

<img width="268" alt="screen shot 2018-09-24 at 2 12 41 pm" src="https://user-images.githubusercontent.com/66834/45936088-b4c5d180-c006-11e8-977d-c360dd7029d8.png">

Whereas in the last release it looked like this:

<img width="256" alt="screen shot 2018-09-24 at 2 33 00 pm" src="https://user-images.githubusercontent.com/66834/45936094-d2933680-c006-11e8-83f0-af4ee213d45e.png">

I tracked this down to the way that MCSS handles the embedded SVGs. They don't cascade in the same way as css stuff. This is because they aren't really css at all, but helpers for compiling data urls. The data url gets compiled at the point you call `mcss.compile` and no more cascading can happen after this.

I managed to fix this by compiling the light/dark with the base theme as a single mcss call. The only alternative is to move all of the SVG stuff into light/dark or base, but that wouldn't be as nice.

Here's the quick and dirty approach I tested:

```diff
diff --git a/styles/light/index.js b/styles/light/index.js
index 6a83b7b4..7ba223df 100644
--- a/styles/light/index.js
+++ b/styles/light/index.js
@@ -4,6 +4,16 @@ var compile = require('micro-css')
 var result = ''
 var additional = ''
 
+var basePath = path.join(__dirname, '..', 'base')
+fs.readdirSync(basePath).forEach(function (file) {
+  if (/\.mcss$/i.test(file)) {
+    result += fs.readFileSync(path.resolve(basePath, file), 'utf8') + '\n'
+  }
+  if (/\.css$/i.test(file)) {
+    additional += fs.readFileSync(path.resolve(basePath, file), 'utf8') + '\n'
+  }
+})
+
 fs.readdirSync(__dirname).forEach(function (file) {
   if (/\.mcss$/i.test(file)) {
     result += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
```
And then removing the appending of base in `styles/index.js`.

I'm sure there is a tidier way to handle this, but hopefully this gives you some ideas!

Reply:
Really appreciate this code, it made resolving the issue *much* easier.